Mathlib/Algebra/Group/Action/Defs.lean

```diff
@@ -71,14 +71,25 @@ lemma op_smul_eq_mul {α : Type*} [Mul α] (a b : α) : MulOpposite.op a • b =
 @[to_additive (attr := simp)]
 lemma MulOpposite.smul_eq_mul_unop [Mul α] (a : αᵐᵒᵖ) (b : α) : a • b = b * a.unop := rfl
 
-/-- Type class for additive monoid actions. -/
+/--
+Type class for additive monoid actions. The `AddAction G P` typeclass says that the
+additive monoid `G` acts additively on a type `P`.
```

Comment:
How about
```suggestion
/--
Type class for additive monoid and additive group actions.
The `AddAction G P` typeclass says that the additive monoid or additive group `G` acts additively on a type `P`.
```
which I think addresses the main confusion, that this _also_ works for group actions.

Reply:
(maybe this is too many "additive"s, you could refer to `+` and `+ᵥ` instead)

Reply:
Really I'm just trying to distinguish the `MulAction` and `AddAction` cases, as we do with `Group` vs `AddGroup`. I would guess that most mathematicians would never write a group action with `+`, and so it makes sense to write a slightly weird docstring here, because the class probably is also weird to them.